src/analyses/ai.h

```diff
@@ -312,6 +312,26 @@ class ai_baset
     const goto_functionst &goto_functions,
     const namespacet &ns);
 
+  // function calls and return are special cases
```

Comment:
Can I take "improve the documentation" along with the point about terminology below as actions for future PRs rather than things that need to be done in this PR?